src/server/scripts/EasternKingdoms/Stratholme/instance_stratholme.cpp

```diff
@@ -95,6 +110,37 @@ class instance_stratholme : public InstanceMapScript
             GuidSet abomnationGUID;
             EventMap events;
 
+            GuidVector TrapGatesGUIDs[2]; //two gate areas
+            std::map<TempSummon*, uint8> PlaguedGatesGUIDs;
```

Comment:
Do not ever store pointers to world objects for longer than "current update tick", if anything causes them to despawn, this will crash

Use ObjectGuid instead